addons/website/static/src/snippets/s_website_form/000.js

```diff
@@ -195,6 +195,15 @@ import { sprintf } from "@web/core/utils/strings";
                 this._disabledStates[inputEl] = inputEl.disabled;
             }
 
+            // Add a tooltip displaying the number of files that can be uploaded
+            // and their size limit.
+            this.el.querySelectorAll('input[type=file]').forEach(inputEl => {
+                const maxFilesNumber = inputEl.dataset.maxFilesNumber;
+                const maxFileSize = inputEl.dataset.maxFileSize;
+                const title = _.str.sprintf(_t("You can choose maximum %s file(s) and each file should not exceed %s MB."), maxFilesNumber, maxFileSize);
+                inputEl.setAttribute('title', title);
+            });
+
```

Comment:
I tried before to set it in the options in `selectDataAttribute` but there were issues to manage that made it less convenient than just adding it at the start. For example: 
- if I do it in `selectDataAttribute`, I need to update it every time I change one of the limits (not really a problem since it's just a `if` but it would be replaced every time we change the value which I think is too much)
- Since it is not set in the XML template, I will need to re-set it every time the field is rerendered. This happens when you hover the "Label Position" option for example, which is something we do really often, when we are moving the mouse towards the Save button for example :)
- And if I set it in the XML template, then I need to add it in the migration script which I find a bit too much for just a tooltip. I am also not sure if it will be translatable if I add it in the migration or if I set it as an attribute in the template :/ Maybe you know ?

Therefore, I thought that just adding it at the start was the simplest solution :)

Reply:
Hmm now that I think of it, adding it in the cleanForSave may be better :thinking: 
I will try :)

Reply:
You could compute it in the clean for save, that would solve the problems you mention AFAIK

I don't see any reason why it would not be translatable :thinking:

Reply:
What a timing :laughing: